src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -1268,12 +1268,9 @@ const TorrentsTable = new Class({
             }
         }
 
-        if (filterTerms) {
-            for (let i = 0; i < filterTerms.length; ++i) {
-                if (name.indexOf(filterTerms[i]) === -1)
-                    return false;
-            }
-        }
+        if ((filterTerms !== undefined) && (filterTerms !== null)
```

Comment:
Just for fun. What's the difference in JS between `undefined` and `null`?

Reply:
`undefined` is used by the system for undeclared/uninitialized values, while `null` is meant to be used by user code.